cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -20,11 +20,17 @@
     EnabledEpoch = 2
 
 [GovernanceSystemSCConfig]
-	ProposalCost = "5000000000000000000" #5 eGLD
-	MinQuorum    = "158000000000000000000" # sqrt(10*NodePrice)
-	MinPassThreshold = "80000000000000000000"
-	MinVetoThreshold = "30000000000000000000"
-    EnabledEpoch = 2
+    NumNodes           = 500
+    ProposalCost       = "5000000000000000000" #5 eGLD
+    ProposalCostV2     = "5000000000000000000" #5 eGLD
+	MinQuorum          = 400
```

Comment:
indentation fail in this section. Please use white spaces instead of tabs (as to align with the rest of the .toml files)

Reply:
Fixed